### PR TITLE
add Integration tests to the toolbox

### LINKED ISSUE #37: implement integration tests in the toolbox
### Summary
We shoudl start to implement integration tests for the different functions in the toolbox. This can be divided in different PRs for each part of the toolbox.

### Next Steps
I propose to divide the integration testing in 3 jobs one for each part of the toolbox:

1. pfm integration that calculates the AUC and one surrogate AUC
2. ev integration 
3. debiasing integration

This way we can paralelize the testing and reduce tha amount of clacs in pfm integration without needing to reduce the number of surrogates for the other pipeplines. Thoughts?

### PR DESCRIPTION
**Pull request by vinferrer**
Closes #37 .

Changes proposed in this pull request:

- add tests for the different workflows in the pipeline pfm,ev and debias
- pfm integration is done only with one ROI reducing the computational cost of testing

### CODE REVIEW

File: connPFM/tests/test_integration.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    )
+    beta_osf = masker.fit_transform(beta_file)
+    fitt_osf = masker.fit_transform(fitt_file)
+    beta_local = masker.fit_transform(
+        join(testpath, f"{basename(bold_file[:-7])}_beta_ETS.nii.gz")
+    )
+    fitt_local = masker.fit_transform(
+        join(testpath, f"{basename(bold_file[:-7])}_fitt_ETS.nii.gz")
+    )
+    np.all(beta_osf == beta_local)
```
Comment (eurunuela) [inline connPFM/tests/test_integration.py:80]: I would make all these a `assert np.closeall()`.

**Review by eurunuela — CHANGES_REQUESTED** [review]
Thanks @vinferrer ! I looks good to me. I'd just make a couple of changes in the way we check the results (see comment).

File: connPFM/tests/test_integration.py (hunk 1 above)
Comment (vinferrer) [reply connPFM/tests/test_integration.py:80]: sure, no problem

**Comment by vinferrer** [comment]
@eurunuela all ready

**Review by eurunuela — APPROVED** [review]
LGTM!

**Pull request merged**